pages/common/git-browse-ci.md

```diff
@@ -0,0 +1,13 @@
+# git browse-ci
+
+> Opens the current `git` repository's CI on websites such as Git.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-browse-ci>.
+
+- Open the current `git` repository's CI configuration on its upstream website:
```

Comment:
```suggestion
- Open the current repository's CI configuration on its upstream website:
```

There is no need to mention that repo is a Git repo because command description highlights that just Git repos are handled.